### ISSUE TITLE
SNES SAMBA config for Analogue Pocket incorrect

### ISSUE THREAD

**Issue by codefaux** [retronas/retronas#389]
After setting everything up (fantastic, love the project) while waiting for my Pocket to arrive later today, I discovered the SNES ROMs aren't exposed via samba.

Following some investigation, `<datadir>/retronas/analoguepocket/Assets/snes` symlinks to `../../../roms/nintendo/superfamicom/msu1` as opposed to (what I assume to be correct) `../../../roms/nintendo/superfamicom`

After correcting it manually, everything looks similar to other devices' layouts. (My Pocket hasn't arrived yet so I don't know if the rest is set up correctly, this is the only fault I've recognized during setup.)

**Comment by sairuk** [comment]
thanks this should be corrected now

**Comment by codefaux** [comment]
I assume you know how to use Linux, if not ping me again and I'll go into more detail.

Actually it's good you asked, I just reinstalled retronas via LXC instead of a full bloated VM and I had to re-apply this. Can confirm, NOT FIXED.


Short version -- This uses /data/retronas -- this is the default, make sure it matches your configuration

- From a console session (remote-viewer, ssh, etc)
```
su - retronas
cd /data/retronas/analoguepocket/Assets/snes
rm ./common
ln -sf ../../../roms/nintendo/superfamicom ./common
```
